newsfragments/2690.feature.rst

```diff
@@ -0,0 +1 @@
+Add sub-stake boost information to staking CLI.
```

Comment:
We should also add a `2690.doc.rst` that provides a newsfragment about documenting staking rewards.